src/styles/themes/default.js

```diff
@@ -66,7 +66,8 @@ const darkTheme = {
     pickerOptionsTextColor: colors.white,
     imageCropBackgroundColor: colors.greenIcons,
     fallbackIconColor: colors.green700,
-    reactionActive: '#003C73',
+    reactionActive: colors.green600,
```

Comment:
NAB: perhaps this would be slightly clearer: 

```suggestion
    reactionActiveBackground: colors.green600,
```

Reply:
Done!